packages/manager/src/features/Linodes/LinodeCreate/Tabs/OperatingSystems.tsx

```diff
@@ -68,6 +85,21 @@ export const OperatingSystems = () => {
           value={field.value}
           variant="public"
         />
+        {imageDeprecatedText && (
+          <Notice
+            dataTestId="os-distro-deprecated-image-notice"
+            spacingBottom={0}
+            spacingTop={16}
+            variant="warning"
+          >
+            <Typography fontFamily={theme.font.bold}>
+              {imageDeprecatedText}. After this date, this OS distribution will
+              no longer receive security updates or technical support. We
+              recommend selecting a newer supported version to ensure continued
+              security and stability for your linodes.
+            </Typography>
+          </Notice>
+        )}
```

Comment:
Rather than having `imageDeprecatedText` be a string, what do you think about having a boolean control this, and putting the content directly in the JSX

```suggestion
        {image && showImageDeprecatedWarning && (
          <Notice
            dataTestId="os-distro-deprecated-image-notice"
            spacingBottom={0}
            spacingTop={16}
            variant="warning"
          >
            <Typography fontFamily={(theme) => theme.font.bold}>
              {image.label} will reach its end-of-life on{' '}
              {formatDate(image.eol ?? '', { format: 'MM/dd/yyyy' })}. After
              this date, this OS distribution will no longer receive security
              updates or technical support. We recommend selecting a newer
              supported version to ensure continued security and stability for
              your linodes.
            </Typography>
          </Notice>
        )}
```

I find that this is a bit cleaner and more readable.